frontend/src/components/Header.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+import IconBrandsGithub from '~icons/fa6-brands/github';
+const props = defineProps<{
+  year?: number;
+  month: number;
+  day?: number;
+  startTime?: number;
+  endTime?: number;
+}>();
```

Comment:
props 를 1개 또는 2개로 묶으면 좋을 것 같아요! 
"2023년 1월"만 받거나 "1월 23일 12:00~12:30"만 있어서?

Reply:
참고로 vueuse의 `useDateFormat`을 사용하면 유용할 것 같아요!
https://vueuse.org/shared/usedateformat/#usedateformat

Reply:
음 일단 useDateFormat을 써 보긴 했는데...쓰기 전이 훨씬 더 입력이 간단하지 않나요? 시간 입력만 받아야 하는 곳에서도 date의 폼 형식을 맞추기 위해 날짜까지 넣어야 하는 거 같아서요

Reply:
@EliseCho13 아마 성현이가 의도한 건 출력을 `useDateFormat`으로 하라고 한 것 같네요!
props로 받아오는 건 number 또는 Date 객체로 받아오는 게 좋을 것 같은데, 개인적인 의견으로는 Date 객체로 받아오면 `year`, `month`, `day` 같은 props는 필요 없이 `startTime`, `endTime` 두 props만 받아와도 돼서 interface가 조금 더 깔끔해질 것 같아요.

```suggestion
const props = defineProps<{
  startTime: Date;
  endTime: Date;
}>();
```

그러면 아래처럼 `useDateFormat`을 사용해서 template에 보여줄 수 있겠죠?

```ts
const startDate = useDateFormat(props.startTime, "M월 D일 HH시 mm분");
const endDate = useDateFormat(props.endTime, "HH시 mm분");
```

```vue
<div>{{ startDate }} ~ {{ endDate }}</div>
```

Reply:
다른 브랜치에 잘못 commit해서...8-header로 풀 해와서 커밋하려 했는데 뭐가 잘못됐는지 수정한 내역이 날아가서 ㅠㅠ 다른 부분은 다 새로 고쳤는데, 그냥 이 부분은 그대로 둔 채로 커밋하겠습니다